Documentation/ApiOverview/Environment/Index.rst

```diff
@@ -34,9 +32,10 @@ Environment PHP API
 getProjectPath()
 ----------------
 
-The environment provides the path to the folder containing the :file:`composer.json`.
-For projects without Composer setup, this is equal to :ref:`Environment-public-path`.
+The environment provides the path to the `top most directory <directory-project>`
+containing the :file:`composer.json`.
```

Comment:
```suggestion
containing the project's :file:`composer.json`.
```

Just to make it clear.

But: legacy installations don't have a project's composer.json.